topological_utils/scripts/LoadPointSet.py

```diff
@@ -7,57 +7,96 @@
 
 from topological_navigation.topological_node import *
 from strands_navigation_msgs.msg import TopologicalNode
+from strands_navigation_msgs.msg import TopologicalMap
+from strands_navigation_msgs.msg import TopMap
+from strands_navigation_msgs.msg import TopNode
+from strands_navigation_msgs.msg import TopEdge
```

Comment:
Almost ready didn't commit it yet as we just changed some definition also I want do something more elegant, than what I have right now. Will commit today, anyway notice is in separate branch as we will test it for a while before releasing